### PR TITLE
Rollbar: Ignore global registry errors

### PR DESCRIPTION
**Pull request by jbirdjavi**

### CODE REVIEW

**Review by soberstadt — APPROVED** [review]

**Review by knutsenm — CHANGES_REQUESTED** [review]
Travis is unhappy - need some gem updates for security

**Comment by jbirdjavi** [comment]
@knutsenm Those changes are in #95 which is still waiting for approval.

**Pull request merged**
